### ISSUE TITLE
Rendering performance degraded even on "Record Mode" "Recording" and without recording

### ISSUE THREAD

**Issue by rse** [exeldro/obs-source-record#38]
Today I've noticed that if I just add Source Record onto a source with something like "Record Mode" == "Recording" and enable it, because of the Record Mode I expected that unless I enable recording in OBS Studio, Source Record does not have any negative effect. But that's now the case: it still dramatically degrades the rendering performance.

It was unnoticed by me until now, as I usually used it just for a single source in most cases. Today I tried to apply it to ISO-record 3 Birddog 4K cameras and once Source Record is enabled (even without hitting recording), the rendering performance degrades from 5ms to 20ms! With recording, it goes even higher (up to 30ms), of course. But what about the difference between 5ms (Source Record disabled) and 20ms (Source Record enabled but still not recording)?

My expectation would be that under "Record Mode" == "Recording", even if Source Record as a filter is enabled, it still does not kick in and at least has no such a dramatic rendering performance degradation effect unless it really does recording. Can you check this, please?

**Comment by exeldro** [comment]
Are replay buffer and stream disabled on the source record filter?

**Comment by rse** [comment]
Yes, both are disabled. I just enabled Recording with "Record Mode" == "Recording" and configured x264 as the encoder plus some settings for x264.

**Comment by ybabts** [comment]
I'm also currently experiencing this issue. If I hide the Source Record filters all at once with a hotkey it will significantly reduce the amount of resources that OBS is taking up.
